doc/instance.md

```diff
@@ -28,7 +30,7 @@ The following arguments are supported:
 - [template](#template) - (Required) Name of template to use for the instance
 - [compute_offering](#compute_offering) - (Required) Name of the compute offering to use for the instance
 - [cpu_count](#cpu_count) - (Required) Number of CPUs the instance should be created with.
-- [memory_in_mb](#memory_in_mb) - (Required) Amount of memory in MB the instance should be created with.
+- [memory_in_mb](#memory_in_mb) - (Required) Amount of memory in MB the instance should be created with. `[512,768,1024,2048,3072,4096,5120,6144,7168,8192,10240,12288,14336,16384,18432,20480,22528,24576]`
```

Comment:
I don't think we should put specific discrete values in the documentation since it could vary from region to region or by offering type.

Reply:
@simongodard I agree, maybe I can leave the first 3 as examples?

Reply:
If it's obvious that they are examples, why not!